### PR TITLE
R4R: Docs for KMS and Ledger User App

### PR DESCRIPTION
**Pull request by chengwenxi**
Docs for https://github.com/irisnet/irishub/pull/1287

### CODE REVIEW

File: docs/software/ledger.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+::: danger
+**Do not lose or share your 24 words with anyone. To prevent theft or loss of funds, it is best to ensure that you keep multiple copies of your mnemonic, and store it in a safe, secure place and that only you know how to access. If someone is able to gain access to your mnemonic, they will be able to gain access to your private keys and control the accounts associated with them.**
+:::
+
+This mnemonic is compatible with IrisNet accounts. The tool used to generate addresses and transactions on the IrisNet network is called `iriscli`, which supports derivation of account keys from a Ledger seed. Note that the Ledger device acts as an enclave of the seed and private keys, and the process of signing transaction takes place within it. No private information ever leaves the Ledger device. 
+
+To use `iriscli` with a Ledger device you will need the following(Since IRISnet is based on cosmos-sdk, the COSMOS app is available for IRISnet):
+
+- [A Ledger Nano with the `COSMOS` app installed and an account](#using-a-ledger-device)
+- [A running `iris` instance connected to the network you wish to use.](./node.md#how-to-start-an-irisnet-network-locally)
```
Comment (wukongcheng) [inline docs/software/ledger.md]: Maybe link the page of `how to join the network` will be better.

File: docs/software/ledger.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- [A `iriscli` instance configured to connect to your chosen `iris` instance.](./cli-client.md)
+
+Now, you are all set to start sending transactions on the network.
+
+At the core of a ledger device, there is a mnemonic used to generate accounts on multiple blockchains (including the IRISNet). Usually, you will create a new mnemonic when you initialize your ledger device.
```
Comment (wukongcheng) [inline docs/software/ledger.md]: IRISnet

File: docs/software/ledger.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+To create an account, you just need to have `iriscli` installed. Before creating it, you need to know where you intend to store and interract with your private keys. The best options are to store them in an offline dedicated computer or a ledger device. Storing them on your regular online computer involves more risk, since anyone who infiltrates your computer through the internet could exfiltrate your private keys and steal your funds.
+
+#### Using a ledger device
+
+::: warning
+**Only use Ledger devices that you bought factory new or trust fully**
+:::
+
+When you initialize your ledger, a 24-word mnemonic is generated and stored in the device. This mnemonic is compatible with IRISNet and IRISNet accounts can be derived from it. Therefore, all you have to do is make your ledger compatible with `iriscli``. To do so, you need to go through the following steps:
```
Comment (wukongcheng) [inline docs/software/ledger.md]: IRISnet

File: docs/software/ledger.md (hunk 1 above)
Comment (chengwenxi) [reply docs/software/ledger.md]: Good idea!

**Pull request merged**